rudder-web/src/main/scala/com/normation/rudder/web/components/RuleEditForm.scala

```diff
@@ -1419,14 +1420,14 @@ class RuleEditForm(
              * Everything is based what has been done for the previous dataTable
              */
         Script(JsRaw(s"""
-            function fnFormatDetail${tabid} ( oTable, nTr ) {
+            function fnFormatDetails${tabid} ( oTable, nTr ) {
```

Comment:
They are on a different scope so there is no problem.

Plus they are not named the same ;) on 636its only fnFormatDetails, here is if suffixed by ${tabid} (0 1 2 3 ...)